app/upgrades/v5/upgrades.go

```diff
@@ -253,3 +258,18 @@ func UpdateIBCDenomTraces(ctx sdk.Context, transferKeeper ibctransferkeeper.Keep
 func equalTraces(dtA, dtB ibctransfertypes.DenomTrace) bool {
 	return dtA.BaseDenom == dtB.BaseDenom && dtA.Path == dtB.Path
 }
+
+// UpdateSlashingParams updates the Slashing params (SignedBlocksWindow) to
+// increase to keep the same wall-time of reaction time, since the block times
+// are expected to be 67% shorter.
+func UpdateSlashingParams(ctx sdk.Context, xk slashingkeeper.Keeper) {
+	params := xk.GetParams(ctx)
+
+	// safety check: make sure the window is still 30000
+	if params.SignedBlocksWindow != 30000 {
+		return
+	}
+
+	params.SignedBlocksWindow = 90000
```

Comment:
qq: why not just multiply the existing value by 3?

Reply:
## TL;DR: 
I considered that... but wanted to make the update explicit.

## Too long:
Only adjust if the window is still 30k... In the event a Param Change gov proposal passed before the upgrade completed, don't do anything.  If the window is 30k, then adjust to new value (in this case 90k because that keeps the wall-time identical)

`params.SignedBlocksWindow *= 3`  is of course functionally equivalent, but requires the reader to know what the current SignedBlocksWindow value is... to compute the new value.   Of course, they can just assume that it is 30k at this point... because that's the only way it is reachable.

Additionally, I don't know if 90k is the new desired amount... there were some murmurs of 75k or 80k.   

At any rate.. uhhhh...  I bet the compiler will optimize the multiply to a constant move 🤣

Reply:
```
evmosd q slashing params --node $TESTNET
downtime_jail_duration: 600s
min_signed_per_window: "0.500000000000000000"
signed_blocks_window: "10000"
slash_fraction_double_sign: "0.050000000000000000"
slash_fraction_downtime: "0.010000000000000000"
```

Looks like this migration won't work for the testnet. I think we need to update it to reflect each corresponding value